apps/storefront-e2e/src/types/user.type.ts

```diff
@@ -1,5 +1,5 @@
-export type TestUserData = {
-  id: string;
+export type TestCustomerData = {
```

Comment:
I think it's worth to rename the file to `customer.type.ts`.

Reply:
I'll fix all of these in the follow-up pr